### PR TITLE
Pharokker integration

### PR DESCRIPTION
**Pull request by mult1fractal**
* integrated pharokka genome annotation
* pharokka parser
  * contigs above 75.00 % completenes by checkV will be plotted
  * filter can be adjusted via the --plot_completeness flag (provide float number like 89.24)
  * parses multi genbank-file to single genbank file
  * then plots these genebankfiles

### CODE REVIEW

**Comment by mult1fractal** [comment]
currently testing the profiles

File: configs/container.config [hunk 1]
```diff
@@ -22,6 +22,7 @@ process {
     withLabel: virsorter        { container = 'multifractal/virsorter:0.1.2' }
     withLabel: phigaro          { container = 'multifractal/phigaro:0.5.2' }
     withLabel: pharokka         { container = 'multifractal/pharokka:v1.7_seqkit' }
+    withLabel: phage_plot       { container = 'nanozoo/pycirclize:3.12--48de646' }
```
Comment (replikation) [inline configs/container.config:25]: i assume node and local config is missing?

**Review by replikation — CHANGES_REQUESTED** [review]
node config and local config needs some love for this i assume

File: configs/container.config (hunk 1 above)
Comment (mult1fractal) [reply configs/container.config:25]: ah i need to remove this.. 
pharokka container is doing this but I need to add them to local and node

**Review by replikation — APPROVED** [review]
only add disks then its fine good job

**Comment by mult1fractal** [comment]
cloud profile also completed w/o error.. merging now

**Pull request merged**